packages/components/psammead-bulletin/src/index.stories.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { Link, LiveLabel } from '@bbc/psammead-story-promo';
+import Image from '@bbc/psammead-image';
+import notes from '../README.md';
+import { Bulletin, BulletinHeading, BulletinCTA, BulletinSummary } from '.';
+
+/* eslint-disable react/prop-types */
+const BulletinComponent = ({ script, service, isLive, dir, type, ctaText }) => {
+  const summaryText = 'This is the summary text';
+  const headlineText = 'This is the headline';
+  const imgSrc =
+    'https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg';
+  const bulletinLink = 'https://bbc.co.uk';
+  const imgAltText = 'Iron man';
+
+  return (
+    <Bulletin>
+      {type === 'video' && <Image src={imgSrc} alt={imgAltText} />}
+      <BulletinHeading script={script} service={service}>
+        <VisuallyHiddenText>{ctaText}, </VisuallyHiddenText>
+        {isLive && (
+          <LiveLabel service={service} dir={dir}>
+            Live
```

Comment:
I'm a little concerned by the hardcoded 'Live' text in english - it's in a couple of places in the codebase and I'd have assumed it should be part of a translation bundle and passed in?

Reply:
Yes it should be a part of translation. This will be passed in from the simorgh configs.